### DISCUSSION TITLE
Katenary V3 - need help to test / report issues

### DISCUSSION THREAD

**Discussion by metal3d** [metal3d/katenary#65] [Announcements]
After a long time changing companies, re-examining priorities in my life, spending time with my family... it's time to keep moving forward on Katernary.

First of all, apologies to all those who have been waiting and asking me when Katenary will result in a version 3. The time seems to be coming 😄 

However, I need your help!

The new version changes drastically, the label names have changed and now systematically propose a YAML format. Also, the management of static files in a tree structure (from a directory) now works, but this requires lengthy testing in your projects.

To all those who are interested:
- test the "devel" branch, if you need help compiling it, or if you want me to make "alpha" versions in the "releases" page, let me know.
- when something doesn't work, or you're missing an option, open an "issue" in GitHub and provide the version you're using (or the Git SHA)

The doc page has changed a lot, and can be found (in "devel" mode) here: https://katenary.readthedocs.io/en/develop/ (I know, the "Basic Usage" page isn't up to date, but the rest is. At least, it should be).

In particular, I've got a real topic concerning automatic testing, and I've thought of some ideas, but it's hard to manage everything on my own. So I'm open to having regular contributors (I'm dreaming, I know).

I'm looking forward to celebrating version 3 with you. I'm taking all the time necessary to offer you a reliable and useful tool, especially useful.

In any case, thank you for your patience, and thank you to my former colleagues who continue to ask me about Katenary - it shows that my tool is being used, that it's useful, and that it potentially has a future.

So, back to your terminals! Go go go!

**Comment by WannaBeGeekster** [comment]
And crickets...  This project seems amazing.  I definitely see a ton of value in it.  Have a huge project I would like to use for testing out the version 3.  Will be installing and testing.  Hopefully you have not been discouraged from moving forward.  I know how time and a new role can derail things.  Will update you on the results.

**Comment by metal3d** [comment]
I'm currently overbooked on several professionnal projects.
I still make some improvements. The "develop" branch is not far a future release :)
